# HG changeset patch
# User jwe@segfault.lan
# Date 1201916861 18000
#      Fri Feb 01 20:47:41 2008 -0500
# Node ID 2b47d821d769f844917240bad6bf4811ab507fbd
# Parent  dece11da64ed86ac2347f48ffa13fd046bef7de3
Added tag ss-2-1-52 for changeset dece11da64ed

diff --git a/.hgtags b/.hgtags
--- a/.hgtags
+++ b/.hgtags
@@ -1,1 +1,2 @@
 d56e0d75e7c9760f2a3e750c2d71a5c52bb76b98 ss-2-1-45
+dece11da64ed86ac2347f48ffa13fd046bef7de3 ss-2-1-52
